test/components/LaunchDarkly.test.js

```diff
@@ -1,36 +1,24 @@
 import React from "react";
-import { shallow } from "enzyme";
-import { Broadcast } from "react-broadcast";
+import { configure, shallow } from "enzyme";
 import LaunchDarkly from "../../src/components/LaunchDarkly";
-import { BROADCAST_CHANNEL } from "../../src/constants/LaunchDarkly";
 
-describe("components/LaunchDarkly", () => {
-  it("sets up a broadcast with the correct channel", () => {
-    const subject = shallow(
-      <LaunchDarkly clientId="080808" user="zeke">
-        <div>Hi</div>
-      </LaunchDarkly>
-    );
-
-    const broadcast = subject.find(Broadcast);
-    expect(broadcast.prop("channel")).toEqual(BROADCAST_CHANNEL);
-  });
+const Adapter = require("enzyme-adapter-react-16");
+configure({ adapter: new Adapter() });
 
+describe("components/LaunchDarkly", () => {
   it("broadcasts the config object", () => {
     const expectedConfig = {
       clientId: "12345",
       user: {
         key: "user123"
       }
     };
-    const subject = shallow(
+    const provider = shallow(
       <LaunchDarkly clientId={expectedConfig.clientId} user={expectedConfig.user}>
         <div>Hi</div>
       </LaunchDarkly>
     );
-
-    const broadcast = subject.find(Broadcast);
-    expect(broadcast.prop("value")).toEqual(expectedConfig);
+    expect(provider.props().value).toEqual(expectedConfig);
```

Comment:
I had to look closely at this test to realize that the `enzyme` update obviated the `find`.  Can we add something to make it clear that we're validating `LaunchDarklyProvider` here?

```
expect(provider).toBeInstanceOf(LaunchDarklyProvider)
expect(provider.props().value).toEqual(expectedConfig);
```

I realize it's not adding anything strictly necessary, but I think would read more clearly.  This test is mostly documentation anyway :)

Reply:
Thanks for your feedback, for my point of view, use enzyme as a test library for React ^16.3.0 it's a little tricky. We could use for example react-testing-library and provide a more detailed test in the future